www/perl-lib/SMT/RegistrationSharing.pm

```diff
@@ -476,7 +476,8 @@ sub _sendData{
     if ($certPath) {
         $ua->setopt(CURLOPT_CAPATH, $certPath);
     }
-    return $ua->post($url, Content=>$data);
+    $ua->post($url, Content=>$data);
+    return;
```

Comment:
But that now returns `undef` instead of the response to the post request, which is checked in many other placed in the code?